lib/jaeger/client/http_sender.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'logger'
+
+module Jaeger
+  module Client
+    class HttpSender
+      def initialize(url:, headers:, encoder:, logger: Logger.new(STDOUT))
```

Comment:
what are the headers in this case? maybe you can also add an example to the readme how to use this adapter.

Reply:
The headers would generally be used for auth with the endpoint. I'll add an example to the readme

Reply:
Ok, but in case they are optional then lets add `: {}`